kong/router/init.lua

```diff
@@ -30,7 +30,137 @@ function _M:select(req_method, req_uri, req_host, req_scheme,
 end
 
 
+
+local phonehome_statistics
```

Comment:
maybe we could move this function into `router/utils.lua`, then we can keep `init.lua` clean